README.md

```diff
@@ -262,6 +262,35 @@ You can test it by running `freshclam` or `freshclam.exe` locally, where you've
 DatabaseMirror http://localhost:8000
 ```
 
+### Use docker
+
+Build docker image
+
+```bash
+docker build . --tag cvdupdate:latest
+```
+
+Run image, that will automaticly update databases in folder `/srv/cvdupdate`
+
+```bash
+docker run -d \
+  -v /srv/cvdupdate:/root/.cvdupdate/database \
+  cvdupdate:latest
```

Comment:
`cvdupdate` in the docker image is running as root, and will create any new files as being owned by the `root` user.  So in this example that works okay if `/srv/cvdupdate` is owned by root.  but if it's a user-owned or service-user owned directory, you'll wind up with some root-owned database files in there.

Do you know if it's possible with docker to have the files owned by the same user as owns the directory?

I imagine also that people won't want to have to the process with elevated root permissions anyhow.

Reply:
Yes, it's possible by adding some additional parameters to `docker` command

Run as current user:

```bash
docker run --user $(id -u) ...
```

Run as user with ID 888:
```bash
docker run --user 888 ...
```

Reply:
Rewrited readme and docker entrypoint.

Now you can add the `USER_ID` environment variable and the application will run as the user with the specified ID

Reply:
Thanks @ismvru I will give it a try!